src/main/java/pl/lukakan/productswebmanager/ProductController.java

```diff
@@ -0,0 +1,52 @@
+package pl.lukakan.productswebmanager;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import java.util.List;
+
+@Controller
+public class ProductController {
+
+    private final ProductRepository productRepository;
+
+    @Autowired
+    public ProductController(ProductRepository productRepository) {
+        this.productRepository = productRepository;
+    }
+
+    @GetMapping("/list")
+    public String showProducts(@RequestParam(name = "kategoria", required = false) String categoryName, Model model) {
```

Comment:
Staraj sie uzywac w urlu wylacznie polskich albo wylacznie angielskich nazw

Reply:
Mozesz tez podawac na przyklad `/list?kategoria=FOOD` i inne odpowiednie wartośiom enuma nazwy, to będziesz mogl tuaj bezposrednio przyjac zmienna typu `Product.Category` zamiast String i przemapowywac to na enuma

Reply:
ooo super dzieki!